packages/web/src/components/apps/panels/settings.tsx

```diff
@@ -1,3 +1,37 @@
+import { Button } from '@srcbook/components/src/components/ui/button';
+import { usePackageJson } from '../use-package-json';
+
 export default function SettingsPanel() {
-  return null;
+  const { status, output, npmInstall } = usePackageJson();
+
+  return (
+    <div className="flex flex-col gap-4 px-5 w-[360px]">
+      <div>
+        <Button onClick={() => npmInstall()} disabled={status === 'installing'}>
+          Run npm install
+        </Button>
+      </div>
+      <div>
+        <Button
+          onClick={() => npmInstall(['uuid'])}
+          variant="secondary"
+          disabled={status === 'installing'}
+        >
+          Run npm install uuid
+        </Button>
+      </div>
+
+      {status !== 'idle' ? (
+        <>
+          <span>
+            Status: <code>{status}</code>
+          </span>
+          <pre className="font-mono text-sm bg-tertiary p-2 overflow-auto rounded-md border">
+            {/* FIXME: disambiguate between stdout and stderr in here using n.type! */}
+            {output.map((n) => n.data).join('\n')}
+          </pre>
+        </>
+      ) : null}
+    </div>
+  );
```

Comment:
Again just wanted to call out here that this is an interim interface! Clicking the two buttons should allow folks to test both the `npm install` and `npm install <packagename>` paths.